vendor/README.md

```diff
@@ -0,0 +1,26 @@
+# Vendor files are populated by the build script:
+
+## What the build script does
+
+Using this repo, run `npm install`... the latest versions of:
+
+* `CodeMirror` (https://github.com/codemirror/CodeMirror) is installed.
+* `jsonlint` (https://github.com/zaach/jsonlint) is installed.
+* `less` (https://github.com/less/less.js) is installed.
+* `lz-string-unsafe` (https://github.com/openstyles/lz-string-unsafe) is installed.
+* `semver-bundle` (https://github.com/openstyles/semver-bundle) is installed.
+* `stylus-lang` (https://github.com/openstyles/stylus-lang-bundle) is installed.<br><br>
+* The necessary build tools are installed; see `devDependencies` in the `package.json`.
+
+## Running the build script
+
+Use `npm run build` to first update the packages in the `node_modules` folder & then update the vendor folder.
+
+The following changes are made:
+
+* `CodeMirror`: Only existing files are updated directly from the `node_modules` folder; see the [CodeMirror readme](codemirror/README) for specifics.
```

Comment:
found one issue here... the link needs to include the extension: `[CodeMirror readme](codemirror/README.md)`

Reply:
you can fix it in the master branch AFAIK

Reply:
ok